src/modules/_nomodule/PixelManipulation.js

```diff
@@ -13,9 +20,24 @@
 module.exports = function PixelManipulation(image, options) {
   // To handle the case where pixelmanipulation is called on the input object itself
   // like input.pixelManipulation(options)
-  
-  const pixelSetter = require('../../util/pixelSetter.js');
-  let wasmSuccess; // Whether wasm succeded or failed
+
+  const isGIF = image.src.includes('gif');
```

Comment:
this is causing the whole error please change it as it was done previously by you

  const imgType = image.src.split(':')[1].split(';')[0].split('/')[1], // MineType of the image -png, jpg, gif etc.
    isGIF = imgType == 'gif';

Reply:
Why is it causing the error? the image src has to be defined. Why is it not defined?